internal/chat/repository/chat_repo_impl.go

```diff
@@ -344,8 +344,38 @@ func (r *ChatRepoImpl) CreateChat(ctx context.Context, chat domain.Chat, tx repo
 }
 
 func (r *ChatRepoImpl) UpdateChat(ctx context.Context, chat domain.Chat) (*domain.Chat, error) {
-	// TODO implement me
-	panic("implement me")
+	query := fmt.Sprintf(`
+	WITH %[1]s AS (
+		UPDATE %[1]s 
+		SET name = $1, image_url = $2
```

Comment:
```suggestion
		SET name = $1, image_url = $2, updated_at = NOW()
```

I think we missed this.